text/20_requirements.md

```diff
@@ -108,7 +108,7 @@ When applied to physical phenomena, it is manifestly clear how silly it would be
 
 ## Functional versus Non-Functional
 
-Functional requirements state what a system should _do_; non-functional requirements state what a system should _be_.
+__Functional requirements__ state what a system should _do_; __non-functional requirements__ state what a system should _be_.
```

Comment:
In the preceding chapters you bolded the terms that were important takeaways, so I did the same here for functional/non-functional requirements.